src/net_processing.cpp

```diff
@@ -2909,13 +2909,13 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             return;
         }
 
-        // We won't accept tx inv's if we're in blocks-only mode, or this is a
+        // Reject tx INVs when the -blocksonly setting is enabled, or this is a
         // block-relay-only peer
-        bool fBlocksOnly = m_ignore_incoming_txs || (pfrom.m_tx_relay == nullptr);
+        bool reject_tx_invs{m_ignore_incoming_txs || (pfrom.m_tx_relay == nullptr)};
```

Comment:
Why not calling it ignore? Reject sounds like something proactive.

Reply:
It will disconnect the peer, which seems proactive and stronger than "ignore"

Reply:
Good point, I forgot it disconnects. Still unsure "reject" captures the meaning tho.
I don't have a better alternative to suggest at this point so feel free to ignore.

Reply:
Happy to change when there is a better name. Resolving for now.